fixture-monkey-javax-validation/src/test/java/com/navercorp/fixturemonkey/javax/validation/spec/BigDecimalIntrospectorSpec.java

```diff
@@ -52,4 +52,12 @@ public class BigDecimalIntrospectorSpec {
 
 	@PositiveOrZero
 	private BigDecimal positiveOrZero;
+
+	@DecimalMax(value = "100.1")
+	@DecimalMin(value = "100.1")
+	private BigDecimal decimalEqual;
+
+	@Max(value = 100)
+	@Min(value = 100)
+	private BigDecimal integerEqual;
```

Comment:
I cannot infer your intent, could you create a new DTO and make a new concise and clear test to reveal your intent?